### PR TITLE
[HZ-1467] Use standard Java binary serialization methods for SQL JSON object

### PR DESCRIPTION
**Pull request by Fly-Style**

### CODE REVIEW

File: hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java [hunk 1]
```diff
@@ -16,131 +16,26 @@
...
-        }
-    }
-
     @Override
     public Object conclude() {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
         try {
-            generator.writeEndObject();
-            generator.close();
+            ObjectOutputStream oos = new ObjectOutputStream(baos);
```
Comment (viliam-durina) [inline hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java:36]: This doesn't write the bytes for the JSON document, but some garbage. Tests fail then.

**Review by viliam-durina — CHANGES_REQUESTED** [review]

File: hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java [hunk 2]
```diff
@@ -16,131 +16,26 @@
...
-            generator = JSON_FACTORY.createGenerator(baos);
-            generator.writeStartObject();
-        } catch (IOException e) {
-            throw sneakyThrow(e);
-        }
-    }
-
     @Override
     public Object conclude() {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
```
Comment (krzysztofslusarski) [inline hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java:34]: Shouldn't it just be:
```java
return ((HazelcastJsonValue) super.conclude()).getValue().getBytes(StandardCharsets.UTF_8);
```
?

**Review by krzysztofslusarski — CHANGES_REQUESTED** [review]

File: hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java (hunk 2 above)
Comment (Fly-Style) [reply hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/inject/JsonUpsertTarget.java]: I am late for 3 minutes :)

**Comment by Fly-Style** [comment]
Need further discussion, closing it.

**Pull request closed without merging**